### PR TITLE
Expire mock configs in $TEST_DATA.

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: hubbot [hunk 1]
```diff
@@ -417,15 +417,17 @@ def expire_work_dirs(github_visible_commits):
...
 
-    keep = { }
+    kept_image_files = { }
+    kept_mock_configs = { }
 
     def keep_image(flavor, os_, tag):
-        keep["%s-%s-*-%s-checksum" % (flavor, os_, tag)] = True
-        keep["%s-%s-*-%s.*" % (flavor, os_, tag)] = True
+        kept_image_files["%s-%s-*-%s-checksum" % (flavor, os_, tag)] = True
+        kept_image_files["%s-%s-*-%s.*" % (flavor, os_, tag)] = True
```
Comment (stefwalter) [inline hubbot:427]: I think we should keep the last two images of a given flavor, in case there's a problem. We can do this by simply adding some more lines here with tag decremented.

File: hubbot [hunk 2]
```diff
@@ -437,21 +439,41 @@ def expire_images():
                     if 'tags' in conf:
                         tags = conf['tags']
                         for os_ in tags:
-                            keep_image (flavor, os_, tags[os_])
+                            keep_image(flavor, os_, tags[os_])
+                            keep_mock(flavor, os_, tags[os_])
                 except:
```
Comment (stefwalter) [inline hubbot]: I think this except in way too broad. Perhaps it should be moved up to the `ast.literal_eval` line?

File: hubbot (hunk 1 above)
Comment (mvollmer) [inline hubbot:427]: (Hmm, tags currently are just strings, without any ordering etc.)  Could we do this in a separate PR?

File: hubbot (hunk 2 above)
Comment (mvollmer) [inline hubbot]: Agreed.

Comment (mvollmer) [inline hubbot]: Done.

File: hubbot (hunk 1 above)
Comment (dperpeet) [inline hubbot:427]: we could sort the files by file creation date or interpret the tags as numbers

**Comment by dperpeet** [comment]
needs rebase

**Comment by mvollmer** [comment]
Rebased.
